gradle/dependencies.gradle

```diff
@@ -6,10 +6,10 @@ buildscript {
   ext.layoutLibPrebuiltSha = 'fa3aa65'
 
   ext.versions = [
-    kotlin: '1.6.10',
+    kotlin: '1.7.0',
```

Comment:
1.7.10 is [out](https://kotlinlang.org/docs/releases.html#release-details), any reason for not using it?

Reply:
When I opened PR, It was blocked by compose compiler version which wasn't stable.
Stable Compiler version was released in 10th August.
https://developer.android.com/jetpack/androidx/releases/compose
https://developer.android.com/jetpack/androidx/releases/compose-kotlin
Currently I updated Kotlin to 1.7.10 and Compose to 1.2.1 with stable compiler version 1.3.0.